src/components/queryExpressionEditor/formMode/FormModeConnected.tsx

```diff
@@ -0,0 +1,17 @@
+import {connect} from 'react-redux';
+import {IReactVaporState, IReduxActionsPayload} from '../../../ReactVapor';
+import {IReduxAction, ReduxUtils} from '../../../utils/ReduxUtils';
+import {selectListBoxOption} from '../../listBox/ListBoxActions';
+import {FormMode, IFormModeDispatchProps, IFormModeProps, IFormModeStateProps} from './FormMode';
+
+const mapStateToProps = (state: IReactVaporState): IFormModeStateProps => {
+    return {
+        expressionEditorsState: state.expressionEditors && state.expressionEditors,

```

Comment:
i don't get it, this is twice the same expression no? why not just `state.expressionEditors` ?